core/node/crypto/config.go

```diff
@@ -48,6 +48,34 @@ const (
 	XChainBlockchainsConfigKey                      = "xchain.blockchains"
 )
 
+var (
+	knownOnChainSettingKeys map[string]string
+	onceInitKeys            sync.Once
+)
+
+func AllKnownOnChainSettingKeys() map[string]string {
```

Comment:
Should we call this from a package init method?

Reply:
then it will be always initialized, even if unused.